src/expandable-section/expandable-section-header.tsx

```diff
@@ -199,11 +214,18 @@ export const ExpandableSectionHeader = ({
     variant,
   };
 
-  const triggerClassName = clsx(styles.trigger, styles[`trigger-${variant}`], expanded && styles['trigger-expanded']);
+  if ((headerDescription || headerCounter || headerInfo || headerActions) && variant !== 'container' && isDevelopment) {
+    warnOnce(
+      componentName,
+      'The `headerCounter`, `headerDescription`, `headerInfo` and `headerActions` props are only supported for the "container" variant.'
+    );
+  }
+
+  const wrapperClassName = clsx(styles.wrapper, styles[`wrapper-${variant}`], expanded && styles['wrapper-expanded']);
```

Comment:
Changing class name of the outer wrapping div from "trigger" to "wrapper" because depending on the case, it is not always the click target anymore.